app/src/main/java/org/wikipedia/usercontrib/UserInformationDialog.kt

```diff
@@ -81,6 +88,22 @@ class UserInformationDialog : DialogFragment() {
         binding.editCount.text = StringUtil.fromHtml(getString(R.string.patroller_tasks_edits_list_user_information_dialog_edit_count_text, editCount))
     }
 
+    private fun sendPatrollerExperienceEvent() {
+        var activity: Context? = context
+        while (activity !is Activity && activity is ContextWrapper) {
+            activity = activity.baseContext
+        }
```

Comment:
Not sure but can we use `requireActivity()` here since this is a `DialogFragment`?